tests/BusinessLogic/Common/TestComponents/Dto/FooDto.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
Is this FooDto necessary? Maybe there are more appropriate name for this test component

Reply:
It's a dummy test component that is here to test different DTO manipulation. We could name it CarDto, StudentDto, MoonDto, it is still the same. Since it is not meant to test any semantics of a specific real object but to test the behavior of a generic component, we left the name FooDto.

Reply:
That's what i thought that is just to test. But the name it's just leading us to confusions. I think you could have found a better name than Foo